debian/updater.go

```diff
@@ -125,26 +126,10 @@ var (
 func (f *Factory) UpdaterSet(ctx context.Context) (driver.UpdaterSet, error) {
 	s := driver.NewUpdaterSet()
 
-	ds, err := f.findReleases(ctx, f.mirror)
```

Comment:
I think we need to keep `findReleases()` or at least call `mkDist()` on the found distributions, otherwise when we're parsing the JSON tracker data and trying to `getDist()` it will return an error (which is ignored currently).

Reply:
ah, good catch